x/gamm/keeper/multihop_test.go

```diff
@@ -94,35 +127,28 @@ func (suite *KeeperTestSuite) TestBalancerPoolSimpleMultihopSwapExactAmountIn()
 				suite.App.PoolIncentivesKeeper.SetDistrInfo(suite.Ctx, test)
 			}
 
-			// Calculate the chained spot price.
-			calcSpotPrice := func() sdk.Dec {
-				dec := sdk.NewDec(1)
-				tokenInDenom := test.param.tokenIn.Denom
-				for i, route := range test.param.routes {
-					if i != 0 {
-						tokenInDenom = test.param.routes[i-1].TokenOutDenom
-					}
-					pool, err := keeper.GetPoolAndPoke(suite.Ctx, route.PoolId)
-					suite.NoError(err, "test: %v", test.name)
-
-					sp, err := pool.SpotPrice(suite.Ctx, tokenInDenom, route.TokenOutDenom)
-					suite.NoError(err, "test: %v", test.name)
-					dec = dec.Mul(sp)
-				}
-				return dec
-			}
-
```

Comment:
Note to reviewer: I got rid of these spot price calculations, it was unclear to me why they were needed for multi-hop tests